### PR TITLE
suggested edits for colored markers and legend

### PR DESCRIPTION
**Pull request by mhines-usgs**
@mwernimont !

### CODE REVIEW

File: js/map.js [hunk 1]
```diff
@@ -1,26 +1,60 @@
...
 mtwsc.bindPopup("<h4>Wyoming-Montana Water Science Center</h4><p>16 Attendees</p>");
 fort.bindPopup("<h4>Fort Collins Science Center</h4><p>29 Attendees</p>");
 
+//Marker locations for future events
+var tampa = L.marker([28.0149678, -82.422698], {icon: redIcon}).addTo(map);
+
+//Popups for future event locations 
+tampa.bindPopup("<h4>Southwest Florida Water Management District</h4><p>Sept 13-15, 2016</p>");
+
+//Group of places trainings have been given already
```
Comment (mhines-usgs) [inline js/map.js]: gah, wait, let me clean this cruft.

**Comment by mwernimont** [comment]
Good?

**Comment by mhines-usgs** [comment]
image

**Comment by mhines-usgs** [comment]
@mwernimont you tell me!

**Comment by mwernimont** [comment]
Looks amazing!!

**Comment by mhines-usgs** [comment]
@mwernimont might want to rethink the title above the map (it's in the past tense!)

**Comment by lindsaycarr** [comment]
@mwernimont  is it easy to have checkboxes to choose what is shown on the plot? And default to upcoming on, past off?

**Comment by mhines-usgs** [comment]
@mwernimont & @lindsaycarr that's doable, like a layer switcher option for the UI?

**Comment by lindsaycarr** [comment]
Sure, that

**Comment by mwernimont** [comment]
I think so, so they have the ability just to see the training coming up or just where have trained, right @lindsaycarr ?

**Comment by lindsaycarr** [comment]
yep, or both

**Pull request merged**